docs/portal/image-registry/viewing-logs.md

```diff
@@ -0,0 +1,10 @@
+# Viewing Logs
+
+ECP keeps a log of your registry projects. The log contains a date- and time-stamped record of the operations performed by users. The following procedure describes how to view logs. To make sure you view the logs for the appropriate project, you can view a project's information before viewing its logs.
+1. In the left pane, click **Image Registry > Registry Projects**.
+2. Perform one of the following tasks:<ul><li>Click a project's **Actions** button and select **Logs**.</ul></li> <ul><li>To view information about a project before viewing its logs, click its name to display the Edit page. To view the project's logs, click the **Logs** button at the top right.</ul></li>
```

Comment:
I'd suggest replacing "Edit page" with "detail page". To users without project update permission, this page is readonly

Reply:
Done.